third_party/gpus/cuda/platform.bzl

```diff
@@ -0,0 +1,39 @@
+CUDA_VERSION = "7.0"
```

Comment:
This has changed from being default of "" to "7.0", which I think is misleading.  Is there a reason why you filled in a default again?